contrib/golang/filters/http/source/golang_filter.cc

```diff
@@ -93,7 +93,7 @@ Http::FilterHeadersStatus Filter::encodeHeaders(Http::ResponseHeaderMap& headers
   // eg. filtermanager may invoke sendLocalReply, when scheme is invalid,
   // with "Sending local reply with details // http1.invalid_scheme" details.
   if (state.state() != FilterState::Done) {
-    ENVOY_LOG(warn,
+    ENVOY_LOG(info,
```

Comment:
info level logs are logs that show up for all users. This should be debug or removed. Same elsewhere.

Reply:
Done, update the logleve to debug.